src/dmd/cond.d

```diff
@@ -24,7 +25,9 @@ import dmd.expression;
 import dmd.expressionsem;
 import dmd.globals;
 import dmd.identifier;
+import dmd.intrange;
```

Comment:
Is this still needed after further changes?

Reply:
No

Reply:
I removed the import and changed a `foreach_reverse` to `foreach`, I was setting the array from right to left and now from left to right, the array result it's the same.